packages/relay-compiler-language-graphitation/tsconfig.json

```diff
@@ -1,15 +1,13 @@
 {
   "compilerOptions": {
-    "composite": false,
+    "composite": true,
```

Comment:
Hmm, don't recall why I changed this but I think there was a reason for it. I assume at least everything builds, so let's see if I run into an issue again.